sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderAsyncClient.java

```diff
@@ -907,7 +969,17 @@
      * @return this object is returned from a successful copyright request.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Copyright> getCopyrightForWorld(ResponseFormat format, IncludeText includeText) {
-        return this.serviceClient.getCopyrightForWorldAsync(format, includeText);
+    public Mono<Response<Copyright>> getCopyrightForWorldWithResponse(boolean includeText) {
+        return this.getCopyrightForWorldWithResponse(includeText, null);
+    }
+
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    Mono<Response<Copyright>> getCopyrightForWorldWithResponse(boolean includeText, Context context) {
+        Mono<Response<Copyright>> responseMono =
```

Comment:
why are you using flatmap here if the function has the same return type as returned by the call to getCopyrightForWorldWithResponseAsync? No need!